bears/vcs/git/GitCommitBear.py

```diff
@@ -6,6 +6,13 @@
 from coalib.results.Result import Result
 
 
+def none_or_bool(value):
+    if value is None:
+        return None
+    else:
+        return bool(value)
```

Comment:
the default isn't type checked why would it, you don't give it as a string, right?